schedule/yast/maintenance/yast_mru_install_minimal_with_addons_aarch64.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: yast_mru_install_minimal_with_addons_aarch64
+vars:
+  PATTERNS: base,enhanced_base
+  YUI_REST_API: 1
+schedule:
+  product_selection:
+    - installation/product_selection/install_SLES
```

Comment:
I don't get this line, yaml schedule based on default we can apply later, I would suggest just not use conditional and have 3 different files reflecting the differences with a list of test modules.
https://progress.opensuse.org/issues/119617

Reply:
As s390x yaml file don't need this test module.
At default yaml file, I set product_selection: []